docs/yuidoc-p5-theme/assets/reference.css

```diff
@@ -0,0 +1,23 @@
+/**
+ * Note: most of the styling for the reference documentation is actually
+ * contained in the p5.js-website repository:
+ *
+ * https://github.com/processing/p5.js-website/blob/master/css/main.css
+ *
+ * However, some of it is located here so we can easily iterate on it
+ * in the same repository.
```

Comment:
The main.css file holds all the css for the whole site. I suppose we could separate it out, but there are some things we want to ensure keep consistent between the reference and the rest of the site, and it seems like this might just end up in duplicated code?